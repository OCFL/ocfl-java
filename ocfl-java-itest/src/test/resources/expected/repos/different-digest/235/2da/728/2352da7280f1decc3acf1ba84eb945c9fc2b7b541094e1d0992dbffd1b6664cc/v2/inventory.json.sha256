ff88a52cea1a1b312447fd9e356deecfe0353eb485ed22fbd77ff0b599a65c9c  inventory.json
