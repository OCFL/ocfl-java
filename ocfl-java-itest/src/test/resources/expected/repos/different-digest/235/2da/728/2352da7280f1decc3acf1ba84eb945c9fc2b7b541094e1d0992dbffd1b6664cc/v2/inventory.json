{
  "id" : "o1",
  "type" : "https://ocfl.io/1.1/spec/#inventory",
  "digestAlgorithm" : "sha256",
  "head" : "v2",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : {
    "2b6e16bbff5690b2272a37ab78d83d7e785ff84964a146212602b25fcc21b299" : [ "v2/content/dir1/file3" ],
    "2cbf24e88a7bb07d6721a9fc9a87a8814b68cef54b576327666001d6b36bc8fc" : [ "v1/content/file2" ],
    "6740a1df50b9d89ea515dc1351ccedd406c4e96c1e7a92f71690d822da16d5fc" : [ "v1/content/file1" ],
    "854435a9412fa3fef6b0ff5c264cdd7f5bd97cde413d7195ac0b242eb8b4d300" : [ "v2/content/file1" ]
  },
  "versions" : {
    "v1" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "commit message",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "2cbf24e88a7bb07d6721a9fc9a87a8814b68cef54b576327666001d6b36bc8fc" : [ "file2" ],
        "6740a1df50b9d89ea515dc1351ccedd406c4e96c1e7a92f71690d822da16d5fc" : [ "file1" ]
      }
    },
    "v2" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "second",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "2b6e16bbff5690b2272a37ab78d83d7e785ff84964a146212602b25fcc21b299" : [ "dir1/file3" ],
        "2cbf24e88a7bb07d6721a9fc9a87a8814b68cef54b576327666001d6b36bc8fc" : [ "file2" ],
        "854435a9412fa3fef6b0ff5c264cdd7f5bd97cde413d7195ac0b242eb8b4d300" : [ "file1" ]
      }
    }
  }
}